app/lib/pre_assembly/from_bundle/file.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module PreAssembly
+  module FromBundle
```

Comment:
why `FromBundle` ?   why not `FromBatch` or `FromBatchContext` or `FromStagingLocation` (now that bundle_dir has been changed to staging_location)